src/pages/Register/index.jsx

```diff
@@ -1,5 +1,47 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import {
+  Agree,
+  AlreadyAc,
+  Button,
+  Checkbox,
+  Container,
+  Element,
+  Form,
+  Heading,
+  Input,
+  Wrapper,
+} from "./styles";
```

Comment:
this is really nice that you have created styled-components

Reply:
Thanks @ivinayakg